src/main/webapp/assets/js/faq-listener.js

```diff
@@ -0,0 +1,21 @@
+
+$(function () {
+    // Listener for FAQ
+    $('.panel').click(
+        function () {
+            if($(this).hasClass("collapsed")){
+                const collapseSign = $('.plusMinusSign');
+                collapseSign.removeClass("fa-minus");
+                collapseSign.addClass("fa-plus");
+                $(this).children("div").children("i").removeClass("fa-plus");
+                $(this).children("div").children("i").addClass("fa-minus");
+                $('.card-toggle').removeClass("card shadow");
+                $(this).parent("div").parent("div").addClass("card shadow");
+            }else{
+                $(this).parent("div").parent("div").removeClass("card shadow");
+                $(this).children("div").children("i").removeClass("fa-minus");
+                $(this).children("div").children("i").addClass("fa-plus");
+            }
+        }
+    );
+});
```

Comment:
Rename this file as `functions.js` and  move the listener to a function ( `$('.panel').cl...` )

Remove the document ready function from here. ( `$(function () {...` )

Call the created function from the `student-ambassador.html` page's ready function.

Reply:
I didn't get it.